dockers/docker-orchagent/enable_counters.py

```diff
@@ -3,11 +3,24 @@
 import swsssdk
 import time
 
+DEFAULT_SMOOTH_INTERVAL = '10'
```

Comment:
can you explain why you decided to use these default values?
are those values used for other counters?

Reply:
any update on this question?

Reply:
RIF/PORT_ALPHA is only only used for rates calculation. ALPHA is calculated based on the smoothing interval which defines the size of the window over which we calculate the average value. In this case I configured the window to be 10s. This way if we have a huge 1s spike in traffic, the average rate value will show a curve descending from the spike to the usual rate over approximately 10s.